include/wx/msw/private/tlwgeom.h

```diff
@@ -113,6 +113,41 @@ class wxTLWGeometry : public wxTLWGeometryBase
             return false;
         }
 
+        if (m_placement.showCmd != SW_SHOWMAXIMIZED && m_placement.showCmd != SW_SHOWMINIMIZED)
+        {
+            RECT rect;
+            ::GetWindowRect(tlw->GetHWND(), &rect);
+            // Height and width should be the same unless the user performed
+            // an Aero Snap operation.
+            if ((rect.bottom - rect.top) !=
+                (m_placement.rcNormalPosition.bottom - m_placement.rcNormalPosition.top)
+                ||
+                (rect.right - rect.left) !=
+                (m_placement.rcNormalPosition.right - m_placement.rcNormalPosition.left))
```

Comment:
Sorry for nitpicking but I think it would be slightly more readable like this:
```suggestion
            const RECT rcNormal = m_placement.rcNormalPosition;
            if ((rcWindow.bottom - rcWindow.top) != (rcNormal.bottom - rcNormal.top) ||
                (rcWindow.right - rcWindow.left) != (rcNormal.right - rcNormal.left))
```
where I also renamed `rect` to `rcWindow` to make it more clear which rect is which and because wxMSW tries (maybe not 100% successfully) to use `rect` for `wxRect` and `rc` for `RECT`.